### PR TITLE
Add uint32 and uint64 kernel support for `Invert`

### PR DESCRIPTION
**Pull request by yongtang**
This fix adds uint32 and uint64 kernel support for `Invert`.

In bitwise_ops.cc, uint32 and uint64 have been registered for `Invert` like other bitwise ops `BitwiseAnd`/`BitwiseOr`/`BitwiseXor`/`LeftShift`/`RightShift`.

However, no uint32 and uint64 kernels available for `Invert` yet.

This fix add uint32 and uint64 kernel for `Invert`, and adds additional test cases to cover the changes.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Comment by drpngx** [comment]
Jenkins, test this please.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by yongtang** [comment]
Ping @caisq @drpngx. Any chance to take a look?

File: tensorflow/core/kernels/cwise_ops_gpu_common.cu.h [hunk 1]
```diff
@@ -136,6 +136,9 @@ struct ApproximateEqual<GPUDevice, T> {
 #define DEFINE_UNARY7(F, T0, T1, T2, T3, T4, T5, T6) \
   DEFINE_UNARY2(F, T0, T1);                          \
   DEFINE_UNARY5(F, T2, T3, T4, T5, T6)
+#define DEFINE_UNARY8(F, T0, T1, T2, T3, T4, T5, T6, T7) \
+  DEFINE_UNARY2(F, T0, T1);                              \
+  DEFINE_UNARY6(F, T2, T3, T4, T5, T6, T7)
```
Comment (caisq) [inline tensorflow/core/kernels/cwise_ops_gpu_common.cu.h]: For efficiency, it is better to use two calls to DEFINE_UNARY_4 here.

**Comment by yongtang** [comment]
@caisq Thanks for the review. The PR has been updated. Please take a look.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by yongtang** [comment]
@caisq The PR has been rebased and pushed to resolve the merge conflict. Please take a look.

**Review by caisq — APPROVED** [review]

**Pull request merged**